e2e_tests/tests/nightly/test_capability.py

```diff
@@ -82,13 +82,3 @@ def test_word_language_lstm_const() -> None:
     config["hyperparameters"]["tied"] = False
 
     exp.run_basic_test_with_temp_config(config, conf.nlp_examples_path("word_language_model"), 1)
-
```

Comment:
Should we comment this out instead of just deleting so we know to return to it eventually?

Reply:
I think this is what git history is for. you don't want a lot of dead commented code laying around.